modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -1904,7 +1904,8 @@ or cornerMinEigenVal.
 CV_EXPORTS_W void goodFeaturesToTrack( InputArray image, OutputArray corners,
                                      int maxCorners, double qualityLevel, double minDistance,
                                      InputArray mask = noArray(), int blockSize = 3,
-                                     bool useHarrisDetector = false, double k = 0.04 );
```

Comment:
The same thing with this function: we need to keep an overload with the original signature.